crates/vm-superio/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "vm-superio"
+version = "0.3.0"
+description = "Emulation for legacy devices"
+keywords = ["superio", "emulation"]
+repository = "https://github.com/rust-vmm/vm-superio"
+readme = "README.md"
```

Comment:
This is not going to be valid anymore when we publish the crate on crates.io because it will not find the README.md file in the root of the crate. The root of the crate is where `Cargo.toml` is defined as I understand it. We should see how we need to address this.

Reply:
I added symlinks similar to how `serde` does it. From what I understand, it won't work with relative paths. [This release](https://crates.io/crates/serde/0.9.15) used relative paths, and it doesn't look right (https://github.com/serde-rs/serde/blob/v0.9.15/serde/Cargo.toml#L12). v1.0.0 used symlinks: https://github.com/serde-rs/serde/tree/v1.0.0/serde, https://crates.io/crates/serde/1.0.0.